src/device-state/current-state.ts

```diff
@@ -64,7 +59,7 @@ const stripDeviceStateInLocalMode = (state: DeviceStatus): DeviceStatus => {
 	};
 };
 
-const report = async ({ stateDiff, conf }: StateReport) => {
+async function report({ stateDiff, conf }: StateReport): Promise<boolean> {
```

Comment:
This makes updating state easier since we now know if the report actually sent something or not.